### PR TITLE
JENKINS-67935: updates compuware plugin permissions

### PR DESCRIPTION
**Pull request by vzamoragbmc**
Permissions for compuware plugins were updated to have only one
developer: cpwr_jenkins

# Description
Updating some permissions for compuware jenkins plugins.
Links to plugins:
- [x] https://github.com/jenkinsci/compuware-ispw-operations-plugin - approved via jira `cpwr_jenkins`
- [x] https://github.com/jenkinsci/compuware-strobe-measurement-plugin approved by [pfhjgd0](https://github.com/pfhjgd0)
- [ ] https://github.com/jenkinsci/compuware-topaz-for-total-test-plugin - needs @RonVorndam or @efhjnw0
- [x] https://github.com/jenkinsci/compuware-zadviser-api-plugin - approved via jira `cpwr_jenkins`

Tagging contributors:
@zhouqr2000 
@VinceCapodanno 
@pfhjgd0
@efhjnw0  
@RonVorndam 
@jgoering41

The current person managing the cpwr_jenkins account is me, so I think my username needs to be added to commiters.
@vzamoragbmc

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [X] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [X] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by vzamoragbmc** [comment]
Unfortunately, none of the maintainers of our plugins work with us anymore, so I don't think they'll be approving this pull request. However, I can refer you to someone in management if you need to do some verifications for the approval process. Thanks!

**Comment by pfhjgd0** [comment]
Please accept this as my approval for the work that Victor is doing. The others do not work and I do not have access to the actual code changes.

**Comment by halkeye** [comment]
We don't really have a policy or procedure for companies reclaiming their plugins. It is pretty clearly compuware plugins.  I don't really want to email an individual, Is there a general support email we can email and confirm with?

**Review by pfhjgd0 — APPROVED** [review]
Approved to change the permissions.

**Comment by timja** [comment]
Can you prove ownership of the `cpwr_jenkins` account please?

You could update the full name on the profile or create / comment on a ticket and post here

**Comment by vzamoragbmc** [comment]
> Can you prove ownership of the `cpwr_jenkins` account please?
> 
> You could update the full name on the profile or create / comment on a ticket and post here
I created this ticket there 2 days ago: https://issues.jenkins.io/browse/JENKINS-67935

Would that be enough proof?

Thanks

**Comment by timja** [comment]
The normal practice for this situation is to wait 2 weeks and if we don't hear back from anyone we can merge it.

I'd suggest separating the approved plugins from the remaining one (https://github.com/jenkinsci/compuware-topaz-for-total-test-plugin)

Then we can proceed with the 3 approved ones now (I've already given you commit access)

**Comment by vzamoragbmc** [comment]
I removed the changes for topaz for total test. Waiting on approval so we can merge. Thanks Tim!

**Pull request merged**
